internal/metastore/model/segment_index.go

```diff
@@ -25,6 +25,7 @@ type SegmentIndex struct {
 	WriteHandoff        bool
 	CurrentIndexVersion int32
 	IndexStoreVersion   int64
+	FinishedUTCTime     uint64
```

Comment:
Does it need to be persisted?

Reply:
Yes, execution time cannot be calculated after the task is finished unless it is saved.